src/Files.App/Views/Properties/HashesPage.xaml

```diff
@@ -134,10 +150,10 @@
 			<!--  Unavailable Label  -->
 			<TextBlock
 				x:Name="UnavailableTextBlock"
-				Margin="16,56,16,16"
+				Margin="16,16,16,16"
```

Comment:
Is this alright?

Reply:
I decided to read the file for each algorithm, so I removed this text block.